client/packages/common/src/ui/components/inputs/Autocomplete/AutocompleteList.tsx

```diff
@@ -80,14 +80,16 @@ export const AutocompleteList = <T,>({
       noOptionsText={noOptionsText}
       onChange={onChange}
       sx={{
-        '& .MuiAutocomplete-listbox': {
-          minHeight: height ? `${height}` : 'auto',
-          maxHeight: height ? `${height}` : 'auto',
-        },
         '& .MuiAutocomplete-inputRoot': {
           width: width ? `${width}px` : 'auto',
         },
       }}
+      ListboxProps={{
```

Comment:
offroad: the customer select modal was looking like this:

<img width="538" alt="Screen Shot 2022-07-26 at 11 38 35 AM" src="https://user-images.githubusercontent.com/9192912/180911679-aefa2fcd-790e-4130-95e1-7fba7c40e7fe.png">

there's a long list of customers, with half of them hidden..